docs/Manual/04 Fusing.md

```diff
@@ -266,15 +267,15 @@ def _():
 
 schedule1 = nest1.create_schedule()
 ```
-The temporary array `C` is used to store the output of `schedule0`, and is then used again as one of the inputs of `schedule1`. Dimensions `i0` and `j0` correspond to the rows and columns of `C` in `schedule0`. Dimensions `i1` and `k1` correspond to the rows and columns of `C` in `schedule1`. Therefore, we fuse `i0` with `i1` and `j0` with `k1`. To do this, we need to correctly line-up the dimensions of the two iteration spaces and perform partial fusing.
+The temporary array `C` stores the output of `schedule0`, which is then used as one of the inputs of `schedule1`. Dimensions `i0` and `j0` correspond to the rows and columns of `C` in `schedule0`. Similarly, dimensions `i1` and `k1` correspond to the rows and columns of `C` in `schedule1`. Therefore, we fuse `i0` with `i1` and `j0` with `k1`. We need to correctly line up the dimensions of the two iteration spaces and perform partial fusing.
 ```python
 schedule1.reorder(i1, k1, j1)
 schedule = acc.fuse((schedule0, schedule1), partial=2)
 f, i, j, k0, j1 = schedule.get_indices()
 ```
-The fused iteration space has a shape of (2, 16, 10, 11, 7): `f` is the fusing dimension, `i` is the result of fusing `i0` and `i1`, `j` is the result of fusing `j0` and `k1`, `k0` is the unfused dimension from `schedule0`, and `j1` is the unfused dimension from `schedule1`. The slice (0, \*, \*, \*, 0) contains a copy of `schedule0` and the slice (1, \*, \*, 0, \*) contains a copy of `schedule1`. The rest of the iteration space is padded with empty elements.
+The fused iteration space has a shape of (2, 16, 10, 11, 7). `f` is the fusing dimension, `i` is the result of fusing `i0` and `i1`, `j` is the result of fusing `j0` and `k1`, `k0` is the unfused dimension from `schedule0`, and `j1` is the unfused dimension from `schedule1`. The slice (0, \*, \*, \*, 0) contains a copy of `schedule0` and the slice (1, \*, \*, 0, \*) contains a copy of `schedule1`. The rest of the iteration space is padded with empty elements.
```

Comment:
I think we can break out the last part of this sentence about k0 and j1 (i.e. about unfused dimensions) in a separate sentence. 

You did that above and it preserved the logical flow while avoiding the long run off sentence, which was nice.